less/svg-radial.less

```diff
@@ -0,0 +1,87 @@
+.radial-progress( @radial-progress-diameter: 76px,
+        @radial-progress-border-size: 5px,
+        @radial-progress-transclude-buffer: 0,
+        @radial-progress-border-color: #8BC53E,
+        @radial-progress-border-base-color: #C3C3C3 ) {
+    
+    @radial-progress-radius: @radial-progress-diameter / 2;
+    @radial-progress-transclude-size: @radial-progress-diameter -
+        @radial-progress-border-size -
+        @radial-progress-border-size -
+        @radial-progress-transclude-buffer -
+        @radial-progress-transclude-buffer;
```

Comment:
Those lines are indented because they are actually being subtracted from the line above